web-client/src/components/figma/BlockStyles.tsx

```diff
@@ -95,6 +96,22 @@ const SettingsListWrapper = styled.div`
   padding: 10px;
   background-color: ${COLORS.greyBackground};
   height: 100vh;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+`;
+
+const SettingsListItemWrapper = styled.div`
+  margin: -25px 0px;
+`;
+
+const SettingsListPopupWrapper = styled.div`
```

Comment:
What I mentioned above for the Item Wrapper applies here too.
I also see here that you used media queries only for adjusting the margin, if this is for the pop-up, then it's good to have it here like this in a separate CSS but generalize it for all popups.
But if this was intended to increase margin so that the delete option appears under the change name option in the list, please remove it and use flex

Reply:
Working on these chnages.